repository/RediStick-Core.package/RsRedisEndpoint.class/instance/keysValueInline..st

```diff
@@ -0,0 +1,10 @@
+private
+keysValueInline: aDictionary
```

Comment:
`keysAndValuesInlined: ` would be a better name

Reply:
done https://github.com/mumez/RediStick/pull/8/commits/e08b474a827594658cfb99de920d091273bb2e30